tests/draft2019-09/optional/ecmascript-regex.json

```diff
@@ -1,11 +1,38 @@
 [
     {
         "description": "ECMA 262 regex non-compliance",
-        "schema": { "format": "regex" },
+        "schema": {
+            "type": "string",
+            "pattern": "^x\\Z"
+        },
+        "tests": [
+            {
+                "description": "\\Z should match literal Z",
```

Comment:
I don't think that `\Z should match literal Z` is always correct, so I would argue that it's not justified here.

`\Z` (and other unsupported flags) behavior differs in ECMA 262 RegExp, depending on the mode.
It's not a valid regexp with `u` flag.

Moreover, other tests here imply that pattern matching should be run in unicode mode, which is incompatible with this test.

Reply:
Can you suggest the change, then? I was going by regex101.com.